requirements.txt

```diff
@@ -39,7 +39,7 @@ MarkupSafe==2.0.1 ; python_version <= '3.10'
 MarkupSafe==2.1.2 ; python_version > '3.10' and python_version < '3.12'
 MarkupSafe==2.1.5 ; python_version >= '3.12'  # (Noble) Mostly to have a wheel package
 num2words==0.5.10 ; python_version < '3.12'  # (Jammy / Bookworm)
-num2words==0.5.13 ; python_version >= '3.12' 
+num2words==0.5.13 ; python_version >= '3.12'
```

Comment:
Please, don't update unrelated lines to your change, this is considered useless diff :-).